src/server/graphql/models/Team/archiveTeam/__tests__/archiveTeam.test.js

```diff
@@ -62,6 +63,7 @@ describe('ArchiveTeam', () => {
     auth0ManagementClient.users.updateAppMetadata.mockReset();
     exchange.publish.mockReset();
     getPubSub.default = () => mockPubSub;
+    tmsSignToken.default = jest.fn(() => 'FAKEENCODEDJWT');
```

Comment:
Looks like you can just do this once at the top of the file, no?

Reply:
concurrently run tests make for sad times if i do that :-(